airflow-core.yaml

```diff
@@ -0,0 +1,136 @@
+package:
+  name: airflow-core
+  version: 2.10.4
+  epoch: 0
+  description: Minimal Airflow core - Platform to programmatically author, schedule, and monitor workflows
+  dependencies:
```

Comment:
Do we need all of these as buildtime deps? bash and busybox and coreutils stood out, but curl etc?

Reply:
Yes curl is in the upstream image, so is mariadb, etc 
```bash
(airflow)find curl
curl
(airflow)
```

Reply:
Even PostgreSQl dev?

Reply:
They seem to have few pg related binaries but seems like I just checked again, I think using `postgres-client` can be enough for them after confirming with apk search:

To name a few:
```bash
(airflow)find psql
psql
(airflow)find pg_dump
pg_dump
(airflow)
```

Reply:
Also to note that upstream has this:
```bash
(airflow)pwd
/lib/postgresql/17/lib
(airflow)ls
pgxs
(airflow)
```
This made me think that we might still PostgreSQL-dev or simple PostgreSQL at least?

Reply:
@mritunjaysharma394 I'd imagine we still need PostgreSQL, but we shouldn't need the dev dependency at runtime